### DISCUSSION TITLE
How-To: Import raw files out of a wget dump or similar website dump

### DISCUSSION THREAD

**Discussion by Juliaria08** [ArchiveBox/ArchiveBox#1387] [Q&A] [answered]
Hello, I've found myself having to download a website from the web archive, and I've got a dump of it, but I don't know how to import it into ArchiveBox.
The tree style is as if you were to have it in a webserver (nginx), but I'd rather not repoint the domain name to localhost and have nginx show the website to itself.
Is there an easier way to import the raw files? (images, photos, videos)?

**Comment by pirate** [comment, marked as answer]
You can technically accomplish this but it's a very labor intensive process right now, as it's not one of our main use-cases. You have to create a `Snapshot` for each URL, and it's recommended to add an `ArchiveResult` entry for each item you want to import.

1. Go to `/add/` and add each URL to create a `Snapshot` entry (you can deselect all the archive methods except `title` to avoid it trying to archive the page on its own)
2. Go to `/admin/core/archiveresult/` and create a new ArchiveResult for each file you want to store for that URL, and point it to your `Snapshot` created earlier for that URL
![image](https://github.com/ArchiveBox/ArchiveBox/assets/511499/c76104f0-8178-4277-adf7-2c80e94c1039)
3. Drag your existing files into the snapshot folder e.g. you can put `mp4` files into `./archive/<timestamp>/media/`, the screenshot into `./archive/<timestamp>/screenshot.png`, etc. (just mimic the structure that archivebox uses [shown here](https://github.com/ArchiveBox/ArchiveBox#output-formats))

You can technically skip step (2), ArchiveBox will leave your files alone inside the snapshot dir, but it wont show them in the UI like it does its own files unless you mimic it's normal structure and add `ArchiveResult` entries. You can also let it run with the default methods in step (1) instead of deselecting them, that way it'll create the Log records for everything automatically, and then you can just overwrite the files on disk that it created.

---

We might support automatically importing existing archives in the future, subscribe to this issue for progress updates: https://github.com/ArchiveBox/ArchiveBox/issues/160
